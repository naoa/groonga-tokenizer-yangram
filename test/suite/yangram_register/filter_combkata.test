register tokenizers/yangram
yangram_register --filter_combkata 1
tokenizer_list
tokenize TokenYaBigramFilterCombkata "今日ハ雨ダ" NormalizerAuto
