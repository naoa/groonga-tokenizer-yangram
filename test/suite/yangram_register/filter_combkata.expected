register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register --filter_combkata 1
[[0,0.0,0.0],"TokenYaBigramFilterCombkata"]
tokenizer_list
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "name": "TokenDelimit"
    },
    {
      "name": "TokenUnigram"
    },
    {
      "name": "TokenBigram"
    },
    {
      "name": "TokenTrigram"
    },
    {
      "name": "TokenBigramSplitSymbol"
    },
    {
      "name": "TokenBigramSplitSymbolAlpha"
    },
    {
      "name": "TokenBigramSplitSymbolAlphaDigit"
    },
    {
      "name": "TokenBigramIgnoreBlank"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbol"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbolAlpha"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbolAlphaDigit"
    },
    {
      "name": "TokenDelimitNull"
    },
    {
      "name": "TokenYaBigramFilterCombkata"
    }
  ]
]
tokenize TokenYaBigramFilterCombkata "今日ハ雨ダ" NormalizerAuto
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0
    },
    {
      "value": "日ハ",
      "position": 1
    },
    {
      "value": "雨ダ",
      "position": 2
    },
    {
      "value": "ダ",
      "position": 3
    }
  ]
]
